src/components/MainContainer/sidebars/SidebarDesktop.tsx

```diff
@@ -11,8 +11,8 @@ import { RootstockLogoIcon } from '@/components/Icons'
 import { useLayoutContext } from '../LayoutProvider'
 import { Tooltip } from '@/components/Tooltip'
 
-const openedWidth = 239
-const closedWidth = 79
+export const openedWidth = 239
```

Comment:
I'm not sure what to here, this is a constant that will be reused, should I move it to constants or keep here as it is right now.

Reply:
foa it should be `OPENED_WIDTH` if it's a constant. As to your question, it's context-specific so should be within it's context